tests/autoyast/installation.pm

```diff
@@ -118,7 +118,8 @@ sub run {
             $num_errors++;
         }
         elsif (match_has_tag('warning-pop-up')) {
-            if (check_screen('warning-partition-reduced', 0)) {
+            # Softfail only on sle, as timeout is there on CaaSP
+            if (check_var('DISTRI', 'sle') && check_screen('warning-partition-reduced', 0)) {
```

Comment:
We don't install opensuse on HDD smaller than in profile, so it's not really expected there.